index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Video App</title>
+  <link rel="stylesheet" href="./index.css" />
+</head>
+
+<body>
+
+  <div class="main">
+
+    <nav class="navbar">
+      <div class="brand">
```

Comment:
Great job done so far. Please i will suggest one line space between line 16 and line 17 to increase code readability.